docs/3 Scaffold a Blockchain/2 Project Scaffold.md

```diff
@@ -0,0 +1,201 @@
+# Project Scaffold Reference
+
+The `starport app` command scaffolds a project. By default, the Cosmos SDK version is Stargate. <!-- what is a project? compared to a "blockchain" or "app" -->
+
+## Address prefix
+
+You can change the way addresses look in your blockchain.
```

Comment:
```suggestion
Account addresses on Cosmos SDK-based blockchains have string prefixes. For example, Cosmos Hub blockchain uses a `cosmos` prefix, so that addresses look like this: `cosmos12fjzdtqfrrve7zyg9sv8j25azw2ua6tvu07ypf`. `cosmos` is the default prefix, but it's very easy to customize it.

When creating a new blockchain, pass a prefix as a value to the `--address-prefix` flag like so:

starport app github.com/hello/planet --address-prefix moonlight

This will make account addresses on you blockchain look like this: `moonlight12fjzdtqfrrve7zyg9sv8j25azw2ua6tvu07ypf`.

To change it manually after the blockchain has been scaffolded, modify the `AccountAddressPrefix` in `app/prefix.go`.
```